src/java/org/pantsbuild/tools/jar/JarBuilder.java

```diff
@@ -289,7 +289,7 @@ protected FileSource(File source) {
       this.source = source;
     }
 
-    public String name() {
+    @Override public String name() {
```

Comment:
The rest of the file is inconsistent in putting @Override on the separate line.  Mostly I was going with the style used for other code near the change.